src/web_app_handler.py

```diff
@@ -42,13 +46,20 @@ def valid_signature(self, body, headers):
             signature = headers["X-Hub-Signature-256"].replace("sha256=", "")
             body_signature = hmac.new(
                 self.config.gh.webhook_secret.encode(),
-                msg=str(body).encode(),
+                msg=body,
```

Comment:
I honestly thought this got fixed in the PR where we fixed the GH webhook secret.
Anyway, I will do a deployment tomorrow with what is on `main` and revert if it's broken.